client/extensions/woocommerce/state/sites/product-categories/selectors.js

```diff
@@ -178,7 +178,11 @@ export const getProductCategoriesLastPage = (
  * @param {Number} [siteId] Site ID to check. If not provided, the Site ID selected in the UI will be used
  * @return {Boolean}       Returns true if currently requesting product categories for a query, excluding all known queried pages.
  */
-export function areProductCategoriesLoadingIgnoringPage( state, query, siteId ) {
+export function areProductCategoriesLoadingIgnoringPage(
+	state,
+	query,
```

Comment:
Mentioned this in the parent PR too, not sure if we want to have `query = {}` here as well.